algebird-core/src/main/scala/com/twitter/algebird/CountMinSketch.scala

```diff
@@ -124,7 +124,11 @@ class CMSMonoid[K: CMSHasher](eps: Double, delta: Double, seed: Int, maxExactCou
   /**
    * Creates a sketch out of multiple items.
    */
-  def create(data: Seq[K]): CMS[K] = data.foldLeft(zero) { case (acc, x) => plus(acc, create(x)) }
+  def create(data: Seq[K]): CMS[K] = {
+    val summation = new CMSSummation(params)
+    data.foreach { k => summation.insert(k, 1L) }
```

Comment:
since this is only one closure for the whole data and since the insert call is already expensive, I'm going to punt on this (since I'm not sure it will actually be faster). I can imagine the jit inlining this.